_posts/languages/python/2000-01-01-start.md

```diff
@@ -44,6 +44,15 @@ The deployment system will use this information to install the last available
 version respecting this constraint. In the previous example Python `3.7.11`
 would be installed.
 
+The `python_version` field only accepts minor versions of Python like `3.10` or
+`3.8`.  If the project need to use a specific Python patch (i.e.: `3.10.1`)
+the Pipfile should contain a `python_full_version` field, i.e.:
```

Comment:
"i.e." means "c'est à dire"
"e.g." means "par exemple"
In this case I think you mean "par exemple", don't you?

```suggestion
The `python_version` field only accepts minor versions of Python like `3.10` or
`3.8`.  If the project needs to use a specific Python patch (i.e.: `3.10.1`)
the Pipfile should contain a `python_full_version` field, e.g.:
```